pkg/cmd/config.go

```diff
@@ -328,6 +328,9 @@ func createAndPrintManifest(printOptions *printOpts) error {
 	case "equinixmetal":
 		cfg.Set(yamled.Path{"cloudProvider", "equinixmetal"}, providerVal)
 		cfg.Set(yamled.Path{"cloudProvider", "external"}, true)
+	case "vmware-cloud-director":
```

Comment:
Please add vmwareCloudDirector to the example config below.

Reply:
Done